app/util/appUtil.js

```diff
@@ -18,14 +18,15 @@ const metadata = {
     LookupSource: 'MultiAddress',
     PeerId: '(Vec<u8>)',
     TokenId: 'u128',
-    TokenMetadata: 'Hash',
+    TokenMetadataKey: '[u8; 32]',
```

Comment:
I think the `32` here should be based off of the env?

Reply:
done